components/uic_log/platform/posix/sl_log.cpp

```diff
@@ -135,22 +147,71 @@ class sl_log_singleton
     }
     sink->set_filter(
       phoenix::bind(&sl_log_filter, severity.or_none(), tag_attr.or_none()));
+
```

Comment:
Is this really needed? The existing logging sink also ends up logging to syslog when run as systemd service.
This would cause log duplication when run as systemd service. The new logging sink for syslog and existing logging sink needs to be kept mutually exclusive, else both sinks would log same message on to syslog with different formatting.
![image](https://github.com/SiliconLabs/UnifySDK/assets/118879678/3bd5d05c-d194-4efa-9c5e-4461f144bbd7)

If the syslog sink is really needed, Please make this mutual exclusive to existing setup, also make it configurable to switch logging setups.

Reply:
This patch was intended for systems without systemd service support.

I agree that it will cause log duplication when systemd is available, and additional log sink should be optionally enabled at compile time with BOOST_LOG_USE_NATIVE_SYSLOG macro. When BOOST_LOG_USE_NATIVE_SYSLOG is not defined, logs will go to UDP port 514.

As rsyslog/syslog-ng are becoming deprecated and replaced with systemd, this patch is not necessary anymore.
I will close this PR, and it can stay as an reference example how to enable logging on legacy systems.